uuidv8.go

```diff
@@ -258,3 +260,32 @@ func (u *UUIDv8) UnmarshalJSON(data []byte) error {
 	*u = *parsed
 	return nil
 }
+
+// Value implements the driver.Value interface for database writes.
+func (u *UUIDv8) Value() (driver.Value, error) {
+	if u == nil || len(u.Node) != 6 {
+		return nil, nil
+	}
+	return ToString(u), nil
+}
+
+// Scan implements the interface for database reads.
+func (u *UUIDv8) Scan(value interface{}) error {
+	switch v := value.(type) {
+	case string:
+		parsed, err := FromString(v)
+		if err != nil {
+			return err
+		}
+		*u = *parsed
+	case []byte:
+		parsed, err := FromString(string(v))
+		if err != nil {
+			return err
+		}
+		*u = *parsed
+	default:
+		return errors.New("unsupported type for UUIDv8")
+	}
+	return nil
```

Comment:
This could be more readable

```suggestion
	switch v := value.(type) {
	case string:
		parsed, err := FromString(v)
		if err != nil {
			return err
		}
		*u = *parsed
	case []byte:
		parsed, err := FromString(string(v))
		if err != nil {
			return err
		}
		*u = *parsed
	}
	return errors.New("unsupported type for UUIDv8")
```